javascript/ql/src/semmle/javascript/security/dataflow/PrototypePollutionCustomizations.qll

```diff
@@ -94,13 +111,16 @@ module PrototypePollution {
       result = TaintedObjectWrapper::label()
     }
 
-    override Dependency getDependency() { result = dependency }
+    override predicate dependencyInfo(string moduleName_, Locatable loc) {
+      moduleName = moduleName_ and
+      location = loc
+    }
   }
 
   /**
    * Holds if `call` is vulnerable to prototype pollution because the callee is defined by `dep`.
    */
-  predicate isVulnerableDeepExtendCall(ExtendCall call, Dependency dep) {
+  predicate isVulnerableVersionOfDeepExtendCall(ExtendCall call, Dependency dep) {
```

Comment:
This think this rename is a breaking change for at least one customer. Can we keep the name as a (deprecated) alias?